keps/sig-node/3983-drop-in-configuration/README.md

```diff
@@ -370,7 +370,8 @@ Fix the invalid configuration, or remove configurations.
 
 - 2023-05-04: KEP initialized.
 - 2023-07-17: Alpha is implemented in 1.28
-- 2023-09-25: KEP targeted to Beta in 1.29
+- 2023-09-25: KEP retargeted to Alpha in 1.29
+- 2024-01-19: Added an [e2e](https://testgrid.k8s.io/sig-node-release-blocking#node-kubelet-serial-containerd&include-filter-by-regex=KubeletConfigDropInDir) test and set KEP target to Beta in 1.30
```

Comment:
do we inherit PRR review here? Are there any updates to the design that needs to be reflected in this documentation?

Reply:
>do we inherit PRR review here?

Do you mean to add a one-liner stating the PRR review for the beta has already been completed in the last cycle?

>Are there any updates to the design that needs to be reflected in this documentation?

All necessary updates were addressed in the previous review.

Reply:
@SergeyKanzhelev PTAL